bowling.js

```diff
@@ -0,0 +1,67 @@
+const frames = 10;
+const rollsPerFrame = 2;
+const totalPins = 10;
+let ball1 = 0;
+let ball2 = 0;
+
+
+class BowlingGame {
+    constructor(frames, rollsPerFrame, totalPins) {
+      this.frames = frames;
+      this.rollsPerFrame = rollsPerFrame;
+      this.totalPins = totalPins;
+      this.scores = [];
+      this.currentFrame = 0;
+    }
+  
+    play() {
+      for (let i = 0; i < this.frames; i++) {
+        this.playFrame();
+      }
+      this.playExtraBalls();
+      console.log("Total Score: ", this.calculateTotalScore());
+    }
+  
+    playFrame() {
+      let ball1 = this.getRandomScore();
+      if (ball1 === this.totalPins) {
+        this.scores[this.currentFrame] = this.totalPins + this.getRandomScore() + this.getRandomScore();
+        this.currentFrame++;
+        return;
+      }
+      let ball2 = this.getRandomScore(this.totalPins - ball1);
+      let frameScore = 0;
+      if (ball1 + ball2 === this.totalPins) {
+        frameScore = this.totalPins + this.getRandomScore();
+      } else {
+        frameScore = ball1 + ball2;
+      }
+      this.scores[this.currentFrame] = frameScore;
+      this.currentFrame++;
+    }
```

Comment:
There's a lot going on in this function, which is understandable as it the playing of the game. It might help to consider either refactoring the function into two, or moving some of the let statements to the top and including some empty lines to improve readability.